Resources/Prototypes/Loadouts/Miscellaneous/survival.yml

```diff
@@ -36,6 +37,23 @@
     species:
     - Snail
 
+# commenting these out because I would like to do emergency vapor boxes at some point.
```

Comment:
these can go in the _impstation folder when they're done but in the interest of getting decapoid adjustments live, we can leave that for a future pr